.github/workflows/main.yml

```diff
@@ -83,7 +83,7 @@ jobs:
         if:   steps.cache-mcuboot.outputs.cache-hit != 'true'  # Install MCUBoot if not found in cache
         run:  |
           cd ${{ runner.temp }}
-          git clone --branch v1.5.0 https://github.com/JuulLabs-OSS/mcuboot
+          git clone --branch v1.7.2 https://github.com/JuulLabs-OSS/mcuboot
```

Comment:
This fixes the following:
![image](https://user-images.githubusercontent.com/5289520/127422505-b001feb4-09d7-4e7f-b8be-83d774f7e740.png)